packages/calling-stateful-client/src/StreamUtilsLogging.ts

```diff
@@ -0,0 +1,234 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { MediaStreamType } from '@azure/communication-calling';
+import { _logEvent } from '@internal/acs-ui-common';
+import { callingStatefulLogger, EventNames } from './Logger';
+
+/**
+ * helper function to manage logging for stream disposals
+ *
+ * @param eventName Name of event that occured when managing streams
+ * @param streamLogInfo Data about the stream in the event
+ * @returns
+ */
+function _logDisposeStreamEvent(
+  eventName: string,
```

Comment:
nit: can we use the enum here as a type?
```suggestion
  eventName: EventNames,
```